Free Ruler/RuleView.swift

```diff
@@ -56,6 +64,28 @@ class RuleView: NSView {
         return NSScreen.screens.first { $0.frame.intersects(window.convertToScreen(frame)) }
     }
 
+    func getUnitLabel() -> String {
+        switch prefs.unit {
+        case .pixels:
+            return "PX"
+        case .millimeters:
+            return "MM"
+        case .inches:
+            return "IN"
+        }
+    }
```

Comment:
Maybe use lowercase spelling? Personally, I think it looks slightly better.

```suggestion
    func getUnitLabel() -> String {
        switch prefs.unit {
        case .pixels:
            return "px"
        case .millimeters:
            return "mm"
        case .inches:
            return "in"
        }
    }
```